qa/SidechainTestFramework/sc_test_framework.py

```diff
@@ -48,25 +49,50 @@ def sync_all(self):
 
     def join_network(self):
         pass
-    
+
     def sc_add_options(self, parser):
         pass
-    
-    #For now it's not active. scutil.generateGenesisData will return None
+
     def sc_generate_genesis_data(self):
-        return generate_genesis_data(self.nodes[0]) #Maybe other parameters in future
-    
+        return None
+
     def sc_setup_chain(self):
-        genesisData = self.sc_generate_genesis_data() #Should interact with mainchain in order to generate Genesis Data for SC Nodes
-        initialize_sc_chain_clean(self.options.tmpdir, 1, genesisData)
+        initialize_sc_chain_clean(self.options.tmpdir, 1)
+
+    """
+    Bootstrap some sidechain nodes.
+    
+    Parameters:
+     - number_of_sidechains_nodes: the number of sidechain nodes to be bootstrapped
+     - number_of_accounts_per_sidechain: an array containing the numbers of accounts to be created for each sidechain node.
+            For example [2, 4, 5] for add 2 accounts to the first node, 4 accounts to the second one and 5 accounts to the third one.
+     - mainchain_node: the mainchain node
     
+    NB: for each account add an amount of 100
+    """
+    def bootstrap_sidechain(self, number_of_sidechains_nodes, number_of_accounts_per_sidechain, mainchain_node):
```

Comment:
Do we need this here? Why not in utils/scutils?

Reply:
Yes we need it here because sidechain test framework extends bitcoin test framework and they have methods for create nodes (mainchain and sidechain) without pass information between them. If we want to extend bitcoin test framework I think we should to keep the same logic of creation and, in case, add other logic such bootstrapping (but inside sidechain test framework).